src/main/java/frc/utils/GBSubsystem.java

```diff
@@ -29,4 +29,6 @@ public final void periodic() {
 
 	protected abstract void subsystemPeriodic();
 
+	public abstract void setBrake(boolean brake);
```

Comment:
If it’s not a must to be in the manager this should not be a must func to override

Reply:
K ill fix it when i can